source/deployment/sso-saml-ldapsync.rst

```diff
@@ -9,13 +9,13 @@ In addition to configuring SAML sign-in, you can optionally configure synchroniz
 To configure SAML synchronization with AD/LDAP:
 
 1. Go to **System Console > Authentication > SAML 2.0** (or **System Console > SAML** in versions prior to 5.12) and set **Enable Synchronizing SAML Accounts With AD/LDAP** to ``true``.
-2. Go to  **System Console > Authentication > AD/LDAP** (or **System Console > AD/LDAP** in versions prior to 5.12) and set **Enable Synchronization with AD/LDAP** to ``true``.
+2. Go to  **System Console > Authentication > SAML 2.0** and set **Ignore Guest Users when Synchronizing with AD/LDAP** to ``true`` or ``false``. 
```

Comment:
The original step 2 is still valid. It should now be step 3.

Reply:
Very good catch, Scott. Thank you for the oversight!

Reply:
Should be resolved with my latest commit. Also updated the ignore step, given its default value.